Gruntfile.js

```diff
@@ -182,7 +182,7 @@ module.exports = grunt => {
         tasks: ['sass'],
       },
       transpile: {
-        files: ['./ts/**/*.ts', './ts/**/*.tsx'],
+        files: ['./ts/**/*.ts', './ts/**/*.tsx', './ts/**/**/*.tsx'],
```

Comment:
Was this really necessary? I thought `/**/*` already meant going recursively into directories.